recipes/graphviz/run_test.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
```

Comment:
We're not planning to create a feedstock for dask right now :-) So please allow us to have it here for a while, until we update the graphviz packages in our default channels.

Then I'll move this test to our dask package :-)